### ISSUE TITLE
Unable to access "resources" file in lab setup; also getting error in step 3-3. Verify Data Pipeline.

### ISSUE THREAD

**Issue by IftesamShk** [aws-samples/aws-analytics-immersion-day#2]
_**Unable to access "resources" file in lab setup**_

Necessary files as per requirements : 
![image](https://user-images.githubusercontent.com/124920547/217913328-1c6aa63a-5058-4c2c-844b-7c1d0b4e0c63.png)

Missing "resources" file while actual deployment : 
![image](https://user-images.githubusercontent.com/124920547/217915394-cee5f6d4-a592-4003-aebf-2d15d4c0ed68.png)

**_Getting error while running below command_**

Command : 

python3 gen_kinesis_data.py --region-name us-west-2 --service-name kinesis --stream-name retail-trans

![image](https://user-images.githubusercontent.com/124920547/217914794-3ea8c415-1d15-47fa-8146-37b5f009ee64.png)

Error : 

[ec2-user@ip-10-0-5-70 ~]$ python3 gen_kinesis_data.py --region-name us-west-2 --service-name kinesis --stream-name retail-trans
Traceback (most recent call last):
  File "gen_kinesis_data.py", line 31, in <module>
    class CustomDatetimeProvider(BaseProvider):
  File "gen_kinesis_data.py", line 32, in CustomDatetimeProvider
    class Meta:
  File "gen_kinesis_data.py", line 34, in Meta
    name: typing.Final[str] = "custom_datetime"
AttributeError: module 'typing' has no attribute 'Final'

![image](https://user-images.githubusercontent.com/124920547/217915009-74a90ea4-91ee-42e1-94eb-38df9dc09d92.png)

**Comment by ksmin23** [comment]
Hi [IftesamShk](https://github.com/IftesamShk)
Thanks for reporting errors

1. Unable to access "resources" file in lab setup
- I updated the synthentic data generator script. So, the new script doesn't require **resources** any more.

2. Getting error while running below command: python3 gen_kinesis_data.py --region-name us-west-2 --service-name kinesis --stream-name retail-trans
- I fixed the error. Let's pull new codes and try it again.
